core/build.sbt

```diff
@@ -1,5 +1,26 @@
 
-libraryDependencies ++= Seq(
-  "org.scalaz" %% "scalaz-core"       % "7.1.2",
-  "org.scalaz" %% "scalaz-concurrent" % "7.1.2"
-)
+val scalazVersion = sys.env.getOrElse("SCALAZ_VERSION", "7.1.11")
+val scalazMajorVersion = scalazVersion.take(3)
+
+libraryDependencies ++= {
+
+  Seq(
+    "org.scalaz" %% "scalaz-core"       % scalazVersion,
+    "org.scalaz" %% "scalaz-concurrent" % scalazVersion
+  )
+}
+
+// Add scalaz version to project version
+version := {
```

Comment:
This convention is a lot more explicit. I significantly prefer it to the `a` version, in so far as this sort of stuff is required.